package.json

```diff
@@ -319,7 +320,48 @@
                         ],
                         "type": "object"
                     }
-                }
+                },
+                "configurationSnippets": [
+                    {
+                        "label": "%AWS.configuration.description.awssam.debug.snippets.lambdaCode.label%",
+                        "description": "%AWS.configuration.description.awssam.debug.snippets.lambdaCode.description%",
+                        "body": {
+                            "type": "aws-sam",
+                            "request": "direct-invoke",
+                            "name": "${3:Invoke Lambda}",
+                            "invokeTarget": {
+                                "target": "code",
+                                "lambdaHandler": "${1:Function Handler}",
+                                "projectRoot": "^\"\\${workspaceFolder}\""
+                            },
+                            "lambda": {
+                                "runtime": "${2:Lambda Runtime}",
+                                "event": {
+                                    "json": []
```

Comment:
Should JSON be an array?

Reply:
? It's just a sample 'empty' event it could be a dict `{}`, an array `[]` or empty string `""` - which do you think is most logical?

Reply:
what would you expect it to be? It's a sample empty event...

Reply:
I think it's expected to be an object rather than an array. That's the format used for the AWS services that send events.

Reply:
So {} ?